pkg/app/cmd.go

```diff
@@ -260,14 +233,14 @@ func (o *options) ShouldTest() bool {
 	return o.test != ""
 }
 
-func (o *options) ArtifactsDir() string {
```

Comment:
we might need to expose this again in the future if this and rundir are decoupled.

Reply:
Agree, but now we should use `artifacts.BaseDir()` (note that we've moved the flag there, invoked by `artifacts.BindFlags()`)